packages/vscode/src/plugins/prisma-language-server/index.ts

```diff
@@ -68,23 +94,18 @@ const plugin: PrismaVSCodePlugin = {
 
     const rootPath = workspace.workspaceFolders?.[0].uri.path
     if (rootPath) {
-      watcher = chokidar.watch(
-        path.join(rootPath, '**/node_modules/.prisma/client/index.d.ts'),
-        {
-          // ignore dotfiles (except .prisma) adjusted from chokidar README example
-          ignored: /(^|[\/\\])\.(?!prisma)./,
-          // limits how many levels of subdirectories will be traversed.
-          // Note that `node_modules/.prisma/client/` counts for 3 already
-          // Example
-          // If vs code extension is open in root folder of a project and the path to index.d.ts is
-          // ./server/database/node_modules/.prisma/client/index.d.ts
-          // then the depth is equal to 2 + 3 = 5
-          depth: 9,
-          // When false, only the symlinks themselves will be watched for changes
-          // instead of following the link references and bubbling events through the link's path.
-          followSymlinks: false,
-        },
-      )
+      const isFileWatcherEnabled = workspace
+        .getConfiguration('prisma')
+        .get('fileWatcher')
+
+      if (isFileWatcherEnabled) {
+        watcher = startFileWatcher(rootPath)
+        console.debug('File Watcher is enabled and started.')
+      } else {
+        console.debug('File Watcher is disabled.')
```

Comment:
Where are those `console.debug` output?

Reply:
Inside the Developer Tools of VS Code
<img width="486" alt="Screen Shot 2021-12-21 at 14 02 54" src="https://user-images.githubusercontent.com/1328733/146934603-0cc8de00-31f1-41d4-a248-d8b69132e64e.png">